kafka/channel/pkg/dispatcher/dispatcher.go

```diff
@@ -76,12 +132,15 @@ type KafkaDispatcherArgs struct {
 
 type consumerMessageHandler struct {
 	sub        subscription
-	dispatcher *eventingchannels.EventDispatcher
+	dispatcher *eventingchannels.MessageDispatcherImpl
 }
 
 func (c consumerMessageHandler) Handle(ctx context.Context, message *sarama.ConsumerMessage) (bool, error) {
-	event := fromKafkaMessage(ctx, message)
-	err := c.dispatcher.DispatchEventWithDelivery(ctx, *event, c.sub.SubscriberURI, c.sub.ReplyURI, &c.sub.Delivery)
+	m := protocolkafka.NewMessageFromConsumerMessage(message)
```

Comment:
Just a clean code nit; could we avoid one letter variable names where possible? Maybe something like `ceMessage` here given that `message` is an overloaded term wrt the sarama consumerMessage?